addons/mail/static/src/models/composer/composer.js

```diff
@@ -87,19 +87,18 @@ function factory(dependencies) {
         /**
          * Post a message in provided composer's thread with given data.
          *
-         * @param {Object} data
+         * @param {Object} [data]
          * @param {string[]} data.attachments
          * @param {*[]} data.canned_response_ids
          * @param {integer[]} data.channel_ids
-         * @param {boolean} [data.isLog=false]
          * @param {string} data.subject
          * @param {integer} [data.subtype_id]
          * @param {string} [data.subtype_xmlid='mail.mt_comment']
          * @param {Object} [options]
          * @param {integer} options.res_id
          * @param {string} options.res_model
          */
-        async postMessage(data, options) {
+        async postMessage(data={}, options={}) {
```

Comment:
I would just remove `data` and `options` entirely. By looking at the code, just `isLog` was passed, and since this PR now turn this info as a field on composer, `postMessage()` should no longer have any parameter.

Reply:
I did that but then I saw that there is a case where you call `postMessage` from `postMessage` but on another composer object passing the data so I was not sure I could removed these arguments

Reply:
If I'm not wrong, this `postMessage()` call with `Object.assign()` and `data` is just a self-call by this method... so it should be safe :)

Reply:
I made another test as the switching is not important to test here, mostly the placeholder. there is already a test in chatter to test the switch